### PR TITLE
[bitnami/rabbitmq] Chart standardised

### PR DESCRIPTION
**Pull request by migruiz4**
**Description of the change**

This PR standardizes the bitnami/rabbitmq chart to be in line with the rest of the catalog.

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the values.yaml and added to the README.md using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by FraPazGal — CHANGES_REQUESTED** [review]
Thanks for the PR! Please take a look at my comments

File: bitnami/rabbitmq/README.md [hunk 1]
```diff
@@ -516,6 +539,35 @@ $ helm upgrade my-release bitnami/rabbitmq --set auth.password=[PASSWORD] --set
 
 | Note: you need to substitute the placeholders [PASSWORD] and [RABBITMQ_ERLANG_COOKIE] with the values obtained in the installation notes.
 
+### To 9.0.0
+
+This major release renames several values in this chart and adds missing features, in order to be aligned with the rest of the assets in the Bitnami charts repository.
+
+- `service.port` has been renamed as ``.
```
[the remark this answers is not in the document]
Comment (migruiz4) [reply bitnami/rabbitmq/README.md]: Done, thanks!

File: bitnami/rabbitmq/templates/configuration.yaml [hunk 2]
```diff
@@ -1,9 +1,12 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ template "rabbitmq.fullname" . }}-config
-  namespace: {{ .Release.Namespace | quote }}
+  name: {{ printf "%s-config" (include "rabbitmq.fullname" .) }}
```
[the remark this answers is not in the document]
Comment (migruiz4) [reply bitnami/rabbitmq/templates/configuration.yaml]: Removed `rabbitmq.fullname` and `rabbitmq.name` helpers

**Review by FraPazGal — APPROVED** [review]
LGTM!

**Pull request merged**